parts/k8s/cloud-init/artifacts/cse_helpers.sh

```diff
@@ -52,6 +52,9 @@ ERR_CIS_ASSIGN_ROOT_PW=111 {{/* Error assigning root password in CIS enforcement
 ERR_CIS_ASSIGN_FILE_PERMISSION=112 {{/* Error assigning permission to a file in CIS enforcement */}}
 ERR_PACKER_COPY_FILE=113 {{/* Error writing a file to disk during VHD CI */}}
 ERR_CIS_APPLY_PASSWORD_CONFIG=115 {{/* Error applying CIS-recommended passwd configuration */}}
+ERR_IOVISOR_KEY_DOWNLOAD_TIMEOUT=116 {{/* Timeout waiting to download IOVisor repo key */}}
```

Comment:
110-120 codes are all for CIS related errors, can you move these new ones to bigger numbers? We try to keep exit codes ordered by theme

Reply:
Is that better? I was wondering why there is a gap 😄

Reply:
Yes, an thank you for adding that comment :D we should really add comments for all of them to clarify at some point...

you need to run make generate to make CI happy